<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>741</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>804</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="newAction"/>
    <addaction name="openAction"/>
    <addaction name="saveAction"/>
    <addaction name="anthorAction"/>
    <addaction name="separator"/>
    <addaction name="printAction"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="undoACtion"/>
    <addaction name="redoAction"/>
    <addaction name="separator"/>
    <addaction name="copyAction"/>
    <addaction name="cutAction"/>
    <addaction name="pasteAction"/>
    <addaction name="selectAction"/>
    <addaction name="separator"/>
    <addaction name="datatimeAction"/>
    <addaction name="fontAction"/>
    <addaction name="colorAction"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="newAction">
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
  </action>
  <action name="openAction">
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
  </action>
  <action name="anthorAction">
   <property name="text">
    <string>另存为(&amp;A)</string>
   </property>
  </action>
  <action name="printAction">
   <property name="text">
    <string>打印(&amp;P)</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>退出()</string>
   </property>
  </action>
  <action name="undoACtion">
   <property name="text">
    <string>撤销(&amp;U)</string>
   </property>
  </action>
  <action name="redoAction">
   <property name="text">
    <string>重做(&amp;R)</string>
   </property>
  </action>
  <action name="copyAction">
   <property name="text">
    <string>复制(&amp;C)</string>
   </property>
  </action>
  <action name="cutAction">
   <property name="text">
    <string>剪切(&amp;X)</string>
   </property>
  </action>
  <action name="pasteAction">
   <property name="text">
    <string>粘贴(&amp;P)</string>
   </property>
  </action>
  <action name="selectAction">
   <property name="text">
    <string>全选(&amp;A)</string>
   </property>
  </action>
  <action name="datatimeAction">
   <property name="text">
    <string>日期(&amp;D)</string>
   </property>
  </action>
  <action name="fontAction">
   <property name="text">
    <string>字体(&amp;F)</string>
   </property>
  </action>
  <action name="colorAction">
   <property name="text">
    <string>颜色(&amp;L)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
